spec/factories.rb

```diff
@@ -1,4 +1,14 @@
 FactoryBot.define do
+
+    factory :plan_recipe do
+        type { "PlanRecipe" }
+        title { "Eggs Benedict" }
+        calories { 200 }
+        time { 45 }
+        meal_type { "breakfast" }
+        user
```

Comment:
recipe also has the link and image, so make sure to add that to the factory

Reply:
On it.

Reply:
Done!